xml/sip_user_agents.xml

```diff
@@ -129,6 +129,36 @@
     <param pos="0" name="os.cpe23" value="cpe:/o:cisco:ios:{os.version}"/>
   </fingerprint>
 
+  <fingerprint pattern="^Cisco-CP(\d{4})/([\d\.]+\d)$">
+    <description>Cisco CP-model IP Phones</description>
+    <example cisco.model="3905" hw.product="CP-3905" os.version="9.4.1">Cisco-CP3905/9.4.1</example>
+    <param pos="1" name="cisco.model"/>
+    <param pos="0" name="hw.vendor" value="Cisco"/>
+    <param pos="0" name="hw.device" value="VoIP"/>
+    <param pos="0" name="hw.product" value="CP-{cisco.model}"/>
+    <param pos="0" name="os.vendor" value="Cisco"/>
+    <param pos="0" name="os.product" value="IOS"/>
```

Comment:
I don' t think these are running IOS but that's what shows up elsewhere in this file. That's something else that can be handled in the PR mentioned above.

Reply:
I've heard QNX might be the OS on some of these Cisco VoIP phones (I had started digging into that, can finish that investigation up for the subsequent PR).